frontend/src/components/dashboard/create/email/EmailTemplate.tsx

```diff
@@ -31,6 +32,19 @@ import { FinishLaterModalContext } from 'contexts/finish-later.modal.context'
 import { saveTemplate } from 'services/email.service'
 import { getCustomFromAddresses } from 'services/settings.service'
 
+const parseFromAddress = (
```

Comment:
should probably go into shared folder now

Reply:
Done!